### PR TITLE
Rename mincut and maxcut to vmin vmax

### PR DESCRIPTION
**Pull request by larrybradley**
### Description

This PR renames the ``min_cut`` and ``max_cut`` keywords in ``simple_norm`` and
``fits2bitmap`` to ``vmin`` and ``vmax``, which are the common names (across other packages) for these keywords. The old names are deprecated.

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

**Comment by larrybradley** [comment]
ping: @astrofrog

File: astropy/visualization/mpl_normalize.py [hunk 1]
```diff
@@ -202,13 +204,15 @@ def inverse(self, values, invalid=None):
         return values_norm * (self.vmax - self.vmin) + self.vmin
 
 
+@deprecated_renamed_argument("min_cut", "vmin", "6.1")
+@deprecated_renamed_argument("max_cut", "vmax", "6.1")
```
Comment (neutrinoceros) [inline astropy/visualization/mpl_normalize.py:208]: This decorator doesn't need to be stacked 
```suggestion
@deprecated_renamed_argument(["min_cut", "max_cut"], ["vmin", "vmax"], ["6.1", "6.1"])
```

File: astropy/visualization/scripts/fits2bitmap.py [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
 from astropy import log
 from astropy.io.fits import getdata
+from astropy.utils.decorators import deprecated_renamed_argument
 from astropy.visualization.mpl_normalize import simple_norm
 
 __all__ = ["fits2bitmap", "main"]
 
 
+@deprecated_renamed_argument("min_cut", "vmin", "6.1")
+@deprecated_renamed_argument("max_cut", "vmax", "6.1")
```
Comment (neutrinoceros) [inline astropy/visualization/scripts/fits2bitmap.py:15]:
```suggestion
@deprecated_renamed_argument(["min_cut", "max_cut"], ["vmin", "vmax"], ["6.1", "6.1"])
```

File: astropy/visualization/scripts/fits2bitmap.py [hunk 3]
```diff
@@ -182,6 +186,18 @@ def main(args=None):
             "(Default is 0.1)."
         ),
     )
+    parser.add_argument(
```
Comment (neutrinoceros) [inline astropy/visualization/scripts/fits2bitmap.py:189]: maybe we could use [mutually exclusive groups](https://docs.python.org/3/library/argparse.html#mutual-exclusion) here to forbid that the new arguments be passes *alongside* their deprecated counterparts.

File: astropy/visualization/scripts/fits2bitmap.py [hunk 4]
```diff
@@ -234,14 +250,24 @@ def main(args=None):
     )
     args = parser.parse_args(args)
 
+    if args.min_cut is not None:
+        warnings.warn('The "--min_cut" argument is deprecated. Use "--vmin" instead.')
+        if args.vmin is None:
+            args.vmin = args.min_cut
```
Comment (neutrinoceros) [inline astropy/visualization/scripts/fits2bitmap.py:256]: Combined with my other suggestion to use mutually exclusive options, the condtional becomes redundant.
```suggestion
        args.vmin = args.min_cut
```

File: astropy/visualization/scripts/fits2bitmap.py [hunk 5]
```diff
@@ -234,14 +250,24 @@ def main(args=None):
...
+
+    if args.max_cut is not None:
+        warnings.warn('The "--max_cut" argument is deprecated. Use "--vmax" instead.')
+        if args.vmax is None:
+            args.vmax = args.max_cut
```
Comment (neutrinoceros) [inline astropy/visualization/scripts/fits2bitmap.py:261]: same as above
```suggestion
        args.vmax = args.max_cut
```

**Review by neutrinoceros — COMMENTED** [review]
Looks good, and I just learned about `@deprecated_renamed_argument`, thank you for that ! a couple minor suggestions I had while reading it

File: astropy/visualization/mpl_normalize.py (hunk 1 above)
Comment (pllim) [reply astropy/visualization/mpl_normalize.py:208]: Huh, didn't know you can use it like that.

Comment (larrybradley) [reply astropy/visualization/mpl_normalize.py:208]: This is news to me too!  Thanks.

File: astropy/visualization/scripts/fits2bitmap.py (hunk 3 above)
Comment (larrybradley) [reply astropy/visualization/scripts/fits2bitmap.py]: Good idea!  I've updated this PR to use mutually exclusive groups.

File: astropy/visualization/scripts/fits2bitmap.py (hunk 4 above)
Comment (larrybradley) [reply astropy/visualization/scripts/fits2bitmap.py]: done

**Comment by larrybradley** [comment]
@neutrinoceros Thanks for the review!  I've implemented your suggestions.

File: astropy/visualization/scripts/fits2bitmap.py [hunk 6]
```diff
@@ -234,14 +253,22 @@ def main(args=None):
     )
     args = parser.parse_args(args)
 
+    if args.min_cut is not None:
```
Comment (pllim) [inline astropy/visualization/scripts/fits2bitmap.py:256]: Do you have to check for the case where both `--min_cut` and `--vmin` are given?

Comment (larrybradley) [reply astropy/visualization/scripts/fits2bitmap.py:256]: tests added, thanks

**Comment by pllim** [comment]
Not sure why CircleCI didn't report back. The runs are green at https://app.circleci.com/pipelines/github/astropy/astropy/19238/workflows/13535080-2160-4133-bc5e-ffd0baf74a88

**Review by neutrinoceros — COMMENTED** [review]
post mortem approval 💀

**Pull request merged**